### PR TITLE
Fixes #72 - all lint errors fixed

### LINKED ISSUE #72: Fix lint

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

File: advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py [hunk 1]
```diff
@@ -185,38 +207,45 @@ def diagnostics_callback(self, msg):
                 if motor.motor_name in status.name and self._prefix.replace("/", "") in status.name:
                     for key_values in status.values:
                         if "Firmware svn revision" in key_values.key:
-                            server_current_modified = key_values.value.split(" / ")
+                            server_current_modified = key_values.value.split(
```
Comment (dg-shadow) [inline advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py]: Why's this line split? surely not too long...

File: advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/__init__.py [hunk 2]
```diff
@@ -1,3 +0,0 @@
-
-
-
```
Comment (dg-shadow) [inline advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/__init__.py:3]: The last three files had loads of lines removed but none put back in as far as I can see... Is that correct?

File: advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py [hunk 3]
```diff
@@ -353,15 +381,14 @@ def save_calib(self):
                                          "no") == 0:
             self.load_calib(filename)
 
-
-
-    def load_calib(self, filename = ""):
+    def load_calib(self, filename=""):
         if filename == "":
```
Comment (dg-shadow) [inline advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py]: shouldn't this be "" == filename?

File: advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py [hunk 4]
```diff
@@ -353,15 +381,14 @@ def save_calib(self):
...
-            filename = QtGui.QFileDialog.getOpenFileName(self._widget, 'Open Calibration', '')
+            filename = QtGui.QFileDialog.getOpenFileName(
+                self._widget, 'Open Calibration', '')
             if filename == "":
```
Comment (dg-shadow) [inline advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py]: ditto

File: advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_library.py [hunk 5]
```diff
@@ -16,7 +16,8 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-import roslib; #roslib.load_manifest('cyberglove')
+import roslib
+# roslib.load_manifest('cyberglove')
```
Comment (dg-shadow) [inline advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_library.py]: How come this line is here and commented out? Surely should have explanation or be removed?

File: advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_mapper.py [hunk 6]
```diff
@@ -22,63 +22,71 @@
 @contact: ugo@shadowrobot.com, contact@shadowrobot.com
 """
 
-import roslib; #roslib.load_manifest('sr_control_gui')
+import roslib
+# roslib.load_manifest('sr_control_gui')
```
Comment (dg-shadow) [inline advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_mapper.py]: ditto

File: advanced/sr_gui_muscle_driver_bootloader/src/sr_gui_muscle_driver_bootloader/bootloader.py [hunk 7]
```diff
@@ -155,38 +174,45 @@ def diagnostics_callback(self, msg):
                 if motor.motor_name in status.name:
                     for key_values in status.values:
                         if "Firmware svn revision" in key_values.key:
-                            server_current_modified = key_values.value.split(" / ")
+                            server_current_modified = key_values.value.split(
+                                " / ")
```
Comment (dg-shadow) [inline advanced/sr_gui_muscle_driver_bootloader/src/sr_gui_muscle_driver_bootloader/bootloader.py]: Why's this split?

File: sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py [hunk 8]
```diff
@@ -416,21 +447,29 @@ def set_reference_grasp(self):
...
+            # We try to activate ethercat hand again (detect if any controllers
+            # are running, and listen to them to extract the current position)
             self.sr_lib.activate_etherCAT_hand()
-            #Some time to start receiving data
+            # Some time to start receiving data
             rospy.sleep(0.5)
-            self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions()
-        
+            self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions(
+            )
```
Comment (dg-shadow) [inline sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py]: dammit lint :( is this better or worse than breaking at the = but needing a" \"? not sure...

File: sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py [hunk 9]
```diff
@@ -439,14 +478,20 @@ def grasp_changed(self):
         interpolate grasps from chosen to current one and from current to chosen
         hand controllers must be running and reference must be set
         """
-        self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions()
-        
+        self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions(
+        )
```
Comment (dg-shadow) [inline sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py]: same

**Comment by ugocupcic** [comment]
@dg-shadow it's not lint, it's just autopep8 (the tool I use for automatic formatting) that goes a bit crazy. Thanks for the comments, I'll fix

File: advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/__init__.py (hunk 2 above)
Comment (ugocupcic) [inline advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/__init__.py:3]: Yes they were deleted (if you look at the folder name, they were in "notUsed", so I thought now was a good time for cleanup).

**Pull request merged**